index.html

```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
             </ul>
           </div>
         </ul>
-      </div>
+      </div>  
 
     </div>
   </div>
 </section>
+
+
+
+
+
+<section class="contact">
+  <div class="bottom">
+<p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<form action="https://formspree.io/f/mlevodye" method="POST" >
+<input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+<input type="email" id="mail" name="user_email" placeholder="Email address" required>
+<textarea type="text" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+<button type="submit" value="submit">Get in touch</button>
+</form>
+  </div>
+</section>
+
+<footer>
```

Comment:
- [ ] As the previous reviewer suggested, there seems to be an extending white space after the footer section on your webpage which is not supposed to be there. Kindly fix this.